.github/workflows/test-and-release-beta.yml

```diff
@@ -50,7 +56,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
```

Comment:
Why do we node Node here as well?

Reply:
perhaps not

Reply:
Yeah I don't see any node related commands after this, seems we don't needed.

Reply:
You were completely right